__tests__/commands/pages/upload.test.ts

```diff
@@ -4,15 +4,19 @@ import nock from 'nock';
 import prompts from 'prompts';
 import { describe, afterEach, it, expect, beforeAll, beforeEach, vi } from 'vitest';
 
-import Command from '../../../src/commands/references/upload.js';
+import GuidesCommand from '../../../src/commands/docs/upload.js';
+import ReferenceCommand from '../../../src/commands/references/upload.js';
```

Comment:
I just updated `references` to be `reference`, including in the file paths in https://github.com/readmeio/rdme/pull/1261
```suggestion
import ReferenceCommand from '../../../src/commands/reference/upload.js';
```